RFC/0000-template.md

```diff
@@ -0,0 +1,91 @@
+# [RFC Template] Title
+
+**Authors:**
+
+- @nickname
+- @nickname
+
+## 1 Executive Summary
+
+*A short paragraph or bullet list that quickly explains what you're trying to do.*
+
+## 2 Motivation
+
+*What motivates this proposal, and why is it important?*
+
+*Here, we aim to get comfortable articulating the value of our actions.*
+
+## 3 Proposed Implementation
+
+*This is the core of your proposal, and its purpose is to help you think through the problem because [writing is thinking](https://medium.learningbyshipping.com/writing-is-thinking-an-annotated-twitter-thread-2a75fe07fade).*
+
+*Consider:*
+
+- *using diagrams to help illustrate your ideas.*
+- *including code examples if you're proposing an interface or system contract.*
+- *linking to project briefs or wireframes that are relevant.*
+
+## 4 Metrics & Dashboards
```

Comment:
"Dashboards"?  I think measurable metrics is fine, and all we need to capture the product success / scope.